example/docker-compose/ingest-storage-single-binary/tempo.yaml

```diff
@@ -0,0 +1,74 @@
+partition_ring_live_store: true
+stream_over_http_enabled: true
+
+server:
+  http_listen_port: 3200
+  log_level: info
+
+distributor:
+  ingester_write_path_enabled: false
+  kafka_write_path_enabled: true
+  receivers:
+    otlp:
+      protocols:
+        grpc:
+          endpoint: "tempo:4317"
+#  log_received_spans:
+#    enabled: true
+#  log_discarded_spans:
+#    enabled: true
+
+compactor:
+  compaction:
+    block_retention: 1h               # overall Tempo trace retention. set for demo purposes
+
+querier:
+  query_live_store: true
+
+metrics_generator:
+  registry:
+    external_labels:
+      source: tempo
+      cluster: docker-compose
+  storage:
+    path: /var/tempo/generator/wal
+    remote_write:
+      - url: http://prometheus:9090/api/v1/write
+        send_exemplars: true
+  traces_storage:
+    path: /var/tempo/generator/traces
+  traces_query_storage:
+    path: /var/tempo/generator/query_traces
+
+query_frontend:
+  rf1_after: "1999-01-01T00:00:00Z"
+
+storage:
+  trace:
+    backend: s3
+    s3:
+      bucket: tempo
+      endpoint: minio:9000
+      access_key: tempo
+      secret_key: supersecret
+      insecure: true
+    wal:
+      path: /var/tempo/wal             # where to store the wal locally
+
+overrides:
+  defaults:
+    metrics_generator:
+      processors: ['span-metrics', 'service-graphs']
+      generate_native_histograms: both
+
+ingest:
+  enabled: true
+  kafka:
+    address: kafka:9092
+    topic:   tempo-ingest
+
+block_builder:
```

Comment:
It is needed to define the default partition?

assigned_partitions:
   block-builder-0: [0]